src/tools/alerts.ts

```
@@ -0,0 +1,242 @@
+      groupBy: z.enum(["signature", "category", "severity", "src", "dst"]).optional().describe("Group results by field"),
+    },
+    async (args) => {
+      try {
+        const alerts = await engine.queryAll<AlertEvent>(
+          ["alert"],
+          (event) => inTimeRange(event.timestamp, args.timeFrom, args.timeTo),
+          { timeRange: { timeFrom: args.timeFrom, timeTo: args.timeTo } },
+        );
+
+        const groupField = args.groupBy ?? "signature";
+        const values = alerts.map((a) => {
```

Biome (2.3.13)

[error] This callback passed to map() iterable method should always return a value.

Add missing return statements so that this callback returns a value on all execution paths.

(lint/suspicious/useIterableCallbackReturn)